wordle-web/src/views/WordleView.vue

```diff
@@ -85,13 +90,25 @@ onUnmounted(() => {
 
 function newGame() {
   overlay.value = true
-  Axios.get('word')
+  let apiPath = 'word'
+  if (route.path == '/wordoftheday') {
+    apiPath = `word/wordoftheday?offsetInHours=${new Date().getTimezoneOffset() / -60}`
+    if (route.query.date) {
+      apiPath += `&date=${route.query.date}`
```

Comment:
I see you're trying to grab the date from the router path here, but I don't think you ever set up the route to accept route parameters? Are you able to actually hit this page with a date and have this work?